contracts/TopShot.cdc

```diff
@@ -83,6 +84,12 @@ pub contract TopShot: NonFungibleToken {
     // Emitted when a Moment is destroyed
     pub event MomentDestroyed(id: UInt64)
 
+    // Emitted when a Moment is locked
+    pub event MomentLocked(id: UInt64, duration: UFix64, expiryTimestamp: UFix64)
```

Comment:
I don't think that these events should be defined or used in the TopShot contract. I think they should be only in the locking contract

Reply:
The reason I defined it here is because in TopShotLocking contract it only knows the UUID of the NFT, but when listening in `nba-api` we need the `TopShot.NFT.id`

Reply:
You can also get the id in there. it is a field on the NFT

Reply:
It is just `nft.id`

Reply:
Thanks, I've moved the events to TopShotLocking!